junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/config/CachingJupiterConfigurationTests.java

```diff
@@ -86,6 +87,17 @@ void cachesParallelExecutionEnabled() {
 		verifyNoMoreInteractions(delegate);
 	}
 
+	@Test
+	void cachesDefaultDisplayNameGeneratorClass() {
+		when(delegate.getDefaultDisplayNameGeneratorClass()).thenReturn(Optional.of(CustomDisplayNameGenerator.class));
+
+		assertThat(cache.getDefaultDisplayNameGeneratorClass()).hasValue(CustomDisplayNameGenerator.class);
+		assertThat(cache.getDefaultDisplayNameGeneratorClass()).hasValue(CustomDisplayNameGenerator.class);
```

Comment:
```suggestion

```

Reply:
This is not by mistake, This Is to test that cache is storing the instance and not calling defaultJupiterConfiguration again.

Reply:
OK. In that regard, please add a comment so that it does not appear to be a copy-and-paste error.

Reply:
acknowledged